### DISCUSSION TITLE
Facing type error when installing

### DISCUSSION THREAD

**Discussion by seng1218** [lshqqytiger/stable-diffusion-webui-directml#280] [Q&A]
Hi guys, I may sound noob but I face this error pop up when i trying to install the webui. 

![Screenshot 2023-09-22 075531](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/55089798/6152994e-6bda-4b96-a4cd-a77764bdd4d2)

is there anything i've done wrong? Hope guidance from masters. TQVM.

**Comment by Cimm-Yeoman** [comment]
I have seem a number of users report this as an error caused by using an older version of Python. I'm no expert myself but successfully have been running the directml web ui. Here's a link for a thread about this on the main stable diffusion web ui, some of these fixes could potentially work for you: [https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/13236](url)
